examples/firehose/selected-namespaces-filter-metric-names-cloudwatch-firehose/main.tf

```diff
@@ -0,0 +1,10 @@
+module "cloudwatch_firehose_coralogix" {
+  source                            = "github.com/coralogix/terraform-coralogix-aws//modules/firehose"
+  firehose_stream                   = var.coralogix_firehose_stream_name
+  privatekey                        = var.coralogix_privatekey
+  include_all_namespaces            = var.include_all_namespaces
+  include_metric_stream_filter      = var.include_metric_stream_filter
```

Comment:
How does it works together?

Reply:
You are asking what will take effect the metric or the namespace filter ?

Reply:
https://github.com/coralogix/terraform-coralogix-aws/blob/2c2f8309b8aa3a907c583e7890e34e858035e593/modules/firehose/main.tf#L375-L376

which was in master, i'll think of something to remove the 'include_all_namespaces' variable's use. Maybe change the way both are named also.